src/containers/AboutUs.js

```diff
@@ -145,15 +144,23 @@ export default function AboutUs() {
       >
         Contributors
       </h5>
-      <p>
-        There are no outside contributors yet! Check out the{" "}
-        <LinkContainer to="/contribution">
-          <Link>
-          <b>Contribute</b>
+      <Grid
+        container
+        direction="row"
+        justifyContent="center"
+        alignItems="center"
+      >
+        <Grid item xs={2}>
```

Comment:
Can grids be done as a mapped list? Probably easier to store contributors' github IDs and names in a list rather than coding directly, will scale better as well

Reply:
Still think this might be a worthwhile change

Reply:
I made the grid dynamic so that, whenever we want to add a contributor, we simply add him/her to the list.